morpheus/app/models.py

```diff
@@ -101,6 +101,12 @@ class Config:
         max_anystr_length = int(1e7)
 
 
+class AttachmentModel(BaseModel):
+    name: str = ...
+    mime_type: str = ...
+    content: bytes = b''
```

Comment:
`content` should be required, also you can use `name: str` instead of `name: str = ...` to indicate the field is required